gateway-ha/src/test/java/io/trino/gateway/ha/handler/TestQueryIdCachingProxyHandler.java

```diff
@@ -61,6 +66,44 @@ public void testExtractQueryIdFromUrl()
                 .isNull();
     }
 
+    @Test
+    public void testQueryIdFromKill()
+            throws IOException
+    {
+        HttpServletRequest req = Mockito.mock(HttpServletRequest.class);
```

Comment:
Rename req to request. Please avoid abbreviations: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-abbreviations